src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java

```diff
@@ -1,43 +1,49 @@
-package com.project.Investment.App.rest.jdbc;
+package com.project.Investment.App.controller.jdbc;
 
 import com.project.Investment.App.dto.EntityDtoRequest;
 import com.project.Investment.App.model.Entity;
 import com.project.Investment.App.service.EntityService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.format.annotation.DateTimeFormat;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
 import java.net.URI;
+import java.time.LocalDate;
 import java.util.List;
 import java.util.Optional;
 
 @RestController
-@RequestMapping("/v1/entity/jdbc/")
+@RequestMapping("/entity")
 public class RestControllerEntityJdbc {
 
     private final EntityService service;
 
     @Autowired
-    public RestControllerEntityJdbc(@Qualifier("entityServiceJdbc") EntityService service) {
+    public RestControllerEntityJdbc(EntityService service) {
         this.service = service;
     }
 
-    @GetMapping("{id}")
-    public Entity getById(@PathVariable(name = "id") String id) {
-        return service.findById(id);
+    @GetMapping("/{id}")
+    public ResponseEntity<List<Entity>> getById(
+            @PathVariable(name = "id") String id,
+            @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @RequestParam(value = "effectiveDate", required = false) Optional<LocalDate> effectiveDate,
+            @RequestParam(value = "limit") Optional<Integer> limit
+    ) {
+        return ResponseEntity.status(HttpStatus.OK).body(service.findById(id, effectiveDate, limit)));
```

Comment:
There is `ResponseEntity.ok()` method designed for this purpose

Reply:
Done